Source/WebCore/bindings/js/JSCustomElementRegistryCustom.cpp

```diff
@@ -202,6 +202,26 @@ JSValue JSCustomElementRegistry::define(JSGlobalObject& lexicalGlobalObject, Cal
     return jsUndefined();
 }
 
+JSValue JSCustomElementRegistry::getName(JSGlobalObject& lexicalGlobalObject, CallFrame& callFrame)
+{
+    VM& vm = lexicalGlobalObject.vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    if (UNLIKELY(callFrame.argumentCount() < 1))
+        return throwException(&lexicalGlobalObject, scope, createNotEnoughArgumentsError(&lexicalGlobalObject));
+
+    auto constructorValue = callFrame.uncheckedArgument(0);
+    if (!constructorValue.isConstructor())
```

Comment:
Do we absolutely need to check it's specifically constructor here? Like if it isn't, `findInterface` will return `nullptr`. We may consider simplifying this check to `wrapped().findInterface(callFrame.uncheckedArgument(0).getObject())` or something.

Reply:
Good point. We can simplify that.